peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx

```diff
@@ -120,14 +122,13 @@ const CollaborationServiceIntegratedView: React.FC = () => {
 
   const handleLangChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
     setLanguage(parseInt(e.target.value));
-    setSyntaxFullLang(e.target.textContent!);
```

Comment:
This code is important, it needs to be in the codebase in order to convey the information about the programming language used. setSyntaxLang only deals with judge0 compatible numbers.